aries-site/src/pages/templates/filtering.mdx

```diff
@@ -0,0 +1,81 @@
+import { Example } from '../../layouts';
+import {
+  FilteringCards,
+  FilteringLists,
+  FilteringWithRangeInput,
+  FilteringWithSelect,
+} from '../../examples';
+
+<Example
+  code="https://raw.githubusercontent.com/hpe-design/design-system/master/aries-site/src/examples/templates/filtering/FilteringCards.js"
+  showResponsiveControls={['laptop', 'mobile']}
+  template
+>
+  <FilteringLists />
+</Example>
+
+## What is filtering?
+
+Filtering allows data to be narrowed down around a set of specified parameters. When users may not have a specific record in mind but are seeking to group data around a certain set of attributes, filtering should be used to allow the user to narrow the data and take action on relevant records.
+
+If a user will likely be looking for a specific record as opposed to groups of data, consider implementing [search](/templates/searching).
+
+## When to use
+
+### Filtering within a List
```

Comment:
Since we are leading the page with a List Filter example, I suggest we reorder these examples to be either: Table, Card, List or Card, Table, List

Reply:
Agreed.